common/static/common/js/discussion/utils.js

```diff
@@ -194,18 +194,6 @@
             params.url = URI(params.url).addSearch({
                 ajax: 1
             });
-            params.beforeSend = function() {
```

Comment:
Is there any test that verifies that a global beforeSend is respected after this change? If not, you should add one.